### ISSUE TITLE
Basic Auth

### ISSUE THREAD

**Issue by bnikolaus** [chrissnell/crabby#3]
I didn't see anything in the docs or code on how to setup basic auth, do you know if this is supported?

**Comment by Atsman** [comment]
@bnikolaus i don't think so, also i don't think that original author will reply. I forked it https://github.com/Atsman/crabby and use in prod at work, we can think about, how to implement it, if u interested

**Comment by chrissnell** [comment]
Not sure how this issue slipped through the cracks but if y'all can get a good PR together for this, I can see about merging it.

**Comment by chrissnell** [comment]
Hey guys.  Sorry for not seeing this sooner.  I am working on this project once again.  I will see about adding basic auth.
